### DISCUSSION TITLE
What is BaseControl in version 7?

### DISCUSSION THREAD

**Discussion by NickCarducci** [uber/react-map-gl#2202] [Q&A]
`Class extends value undefined is not a constructor or null`

````
class Marker extends BaseControl {
  _render() {
  }
}
````
I believe this marker-feature is important for the [supercluster](https://github.com/visgl/react-map-gl/discussions/1377).

**Comment by Pessimistress** [comment]
There are no class components in v7. Use https://visgl.github.io/react-map-gl/docs/api-reference/use-control
